packaging/os/homebrew_cask.py

```diff
@@ -503,8 +525,13 @@ def main():
     if state in ('absent', 'removed', 'uninstalled'):
         state = 'absent'
 
+
+    p['install_options'] = p['install_options'] or []
+    install_options = ['--{0}'.format(install_option)
```

Comment:
Right you are, thanks for catching this!
The examples need to be fixed. Code is fine though.